.github/workflows/deploy.yml

```diff
@@ -0,0 +1,62 @@
+name: Deploy to dev and prod
+on:
+  push:
+    branches:
+      - master
+env:
+  IMAGE: docker.pkg.github.com/${{ github.repository }}/sykmeldinger-backend:${{ github.sha }}
+
+jobs:
+  build:
+    name: Checkout code and create docker tag
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup java
+        uses: actions/setup-java@v1
+        with:
+          java-version: '12.0.2'
+      - name: Checkout code
+        uses: actions/checkout@master
```

Comment:
Gjerne legg til denne:
      - name: Gradle wrapper validation
        uses: gradle/wrapper-validation-action@v1

Reply:
Hva gjør den? Bruker vi den noe annet sted? :)

Reply:
For andre som lurer: https://blog.gradle.org/gradle-wrapper-checksum-verification-github-action